src/Modules/Manage/Dnn.PersonaBar.Users.Tests/CommandTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Dnn.PersonaBar.Library.Prompt;
+using DotNetNuke.Entities.Portals;
+using Dnn.PersonaBar.Library.Prompt.Models;
+using DotNetNuke.Entities.Users;
+
+namespace Dnn.PersonaBar.Users.Tests
+{
+    public abstract class CommandTests
```

Comment:
CommandTest will help to achieve DRY for existing tests as couple of properties and methods are used repeatedly like PortalSettings, ErrorModelResult, GetUserInfo and also good to abstract the "Act" part for these test as more or less it is same except for the arguments and type of ConsoleCommand i.e RestoreUser, GetUser etc

Reply:
Good idea. May I suggest making it generic

```c#
public abstract class CommandTests<TCommand> where TCommand : ConsoleCommandBase
{
}
```